supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -570,6 +620,58 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:  xref:xplat[Microsoft Azure Cross-Platform Command-Line Interface]
 
+[discrete]
+[[ARM64]]
+==== image:images/yes.png[yes] ARM64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this format for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use this format in general sentences when referring to OCP features.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Creating an ARM64 compute machine set._ 
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*:
+
+[discrete]
+[[arm64]]
+==== image:images/yes.png[yes] arm64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this format for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use lowercase (and backticks) when referring to objects or parameters.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Valid values are `arm64`._
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*:
+
+[discrete]
+[[64-bit ARM]]
+==== image:images/yes.png[yes] 64-bit ARM (noun)
+*Description*: Use in general cases when writing for both names of the architectures for various cloud providers. The term 64-bit ARM systems can refer to AArch64 and ARM64/`arm64`. Using this format can avoid confusion for users when the writer wants to write a general description of the architectures. 
```

Comment:
```suggestion
*Description*: A 64-bit version of the ARM architecture. This term can refer to both AArch64 and ARM64/`arm64`. Use this format in general cases when writing for both names of the architectures for various cloud providers. 
```
Does the "using this format" section add any additional value or describe a new use case? If the latter, incorporate into the "Use this format in general cases..." sentence.

Reply:
Its mostly to introduce the term, but I like your suggestion. Thank you!